cmd/antrea-agent/agent.go

```diff
@@ -129,5 +129,6 @@ func run(o *Options) error {
 
 	<-stopCh
 	klog.Info("Stopping Antrea agent")
+	klog.Flush()
```

Comment:
I think to move this line to the beginning of this function with `defer` would better.

Reply:
Have removed this line. I think we can just use flush when there is os.exit() case. Otherwise, defer will be executed.